drivers/include/saul.h

```diff
@@ -91,6 +91,7 @@ enum {
     SAUL_SENSE_PRESS    = 0x89,     /**< sensor: pressure */
     SAUL_SENSE_ANALOG   = 0x8a,     /**< sensor: raw analog value */
     SAUL_SENSE_UV       = 0x8b,     /**< sensor: UV index */
+    SAUL_SENSE_RADTEMP  = 0x8d,     /**< sensor: radient temperature */
```

Comment:
I'm not convinced that this makes sense, right now, IMHO its still a temperature reading. Adding meta data about type/object etc of the measurement would open up to possibly define lots of different _temperatures_. @haukepetersen what's your opinion as the SAUL inventor and expert?

Reply:
saul_driver_t has three elements: (1) read function, (2) write function, and (3) type. Given that "type" is used for SAUL to identify a sensor (saul_reg_find_type function),  I think "type" should distinguish radiant temperature from air temperature. This modification is to represent that "type". Without using a different type, SAUL detects only one of multiple temperature sensors. Waiting for @haukepetersen's opinion.

Reply:
+1. The spelling in the comment is wrong (radient).

Reply:
@kaspar030, fixed. Thanks! 👍